src/utils/flow.js

```diff
@@ -1,16 +1,25 @@
 /* @flow */
 import type { FlowType, FlowTypes } from "./types";
 
-function flowTypeAnnotationToString(type: TypeTypeAnnotation, nullable: boolean): FlowType {
+function flowTypeAnnotationToString(type: TypeTypeAnnotation, nullable: boolean, variables?: Object): FlowType {
   switch (type.type) {
     case "BooleanTypeAnnotation":
-      return { type: "boolean", nullable };
+      return { type: "boolean", nullable, variables };
     case "NumberTypeAnnotation":
-      return { type: "number", nullable };
+      return { type: "number", nullable, variables };
     case "StringTypeAnnotation":
-      return { type: "string", nullable };
+      return { type: "string", nullable, variables };
+    case "StringLiteralTypeAnnotation":
+    case "BooleanLiteralTypeAnnotation":
+    case "NumericLiteralTypeAnnotation":
+      return {
+        type: "literal",
+        nullable,
+        variables,
+        value: typeof type.value === "string" ? type.value.replace(/'/g, "\"") : type.value
+      };
```

Comment:
This is a good question. If you happen to have a mismatch, there is a error that gets thrown. I had temporarily replaced the `1` with `"'1'"` and it threw the following (which is why I thought it might be okay to let the original babel plugin do some of the work):

```
File:  /Users/kalley/src/babel-plugin-flow-relay-query/test/fixture/fragment/variables/source.js
Error: Argument "first" has invalid value "1".
Expected type "Int", found "1".
```